cmd/tableprinters/image.go

```diff
@@ -38,11 +35,7 @@ func (t *TablePrinter) ImageTable(data []*models.V1ImageResponse, wide bool) ([]
 			usedBy = strings.Join(image.Usedby, "\n")
 		}
 
-		if wide {
-			rows = append(rows, []string{id, name, description, features, expiration, status, usedBy})
-		} else {
-			rows = append(rows, []string{id, name, description, features, expiration, status})
-		}
+		rows = append(rows, []string{id, name, description, features, expiration, status, usedBy})
```

Comment:
The old version was showing the UUIDs on `-o wide`: https://github.com/metal-stack/metalctl/blob/v0.11.9/cmd/output/printer.go#L946-L948